src/main/java/it/gov/pagopa/afm/calculator/model/calculator/Transfer.java

```diff
@@ -13,7 +12,7 @@ public class Transfer implements Comparable<Transfer> {
     private Long taxPayerFee;
     private long primaryCiIncurredFee;
     private PaymentMethod paymentMethod;
-    private Touchpoint touchpoint;
+    private String idTouchpoint;
```

Comment:
the response field name is always `touchpoint`. Furthermore we should return touchpoint mnemonic name not the id